common/lib/client/realtimechannel.js

```diff
@@ -136,50 +161,53 @@ var RealtimeChannel = (function() {
 			}
 		}
 		if(flags) {
```

Comment:
Now we have a properly-spec'd api for setting flags (or at least, will do soon) we're going to want to deprecate calling attach() with an array of flags. I don't believe we ever told any customers about it, and certainly didn't document it -- it was mainly for use by realtime tests -- but still probably worth doing a release or two with it working but printing loud warnings to the console about contacting us, in case anyone discovered it on their own

Reply:
Fixed already.